app/externalapi.py

```diff
@@ -0,0 +1,50 @@
+
+import requests
+
+def headlines_feed():
+
+    query_params = {
+        "sources": "bbc-news,the-verge",
+        "apiKey": "e90358389f0b41b2bf25bfc55c92f69c"
+    }
+
+    main_url = "https://newsapi.org/v2/top-headlines"
```

Comment:
I would like to see less duplication here, which means you could probably extract the route to the API ("https://newsapi.org/v2") into a constant (inside this class, not somewhere else, as it's relevant only here) and reuse it in all 3 functions.